ipn/ipnlocal/local.go

```diff
@@ -1829,6 +1824,14 @@ func mutationsAreWorthyOfTellingIPNBus(muts []netmap.NodeMutation) bool {
 	return false
 }
 
+func (b *LocalBackend) pickNewAutoExitNode() {
+	unlock := b.lockAndGetUnlock()
+	defer unlock()
+	b.setAutoExitNodeIDLockedOnEntry(unlock)
+
+	// TODO(bradfitz,adrian): need to send an ipn.Notify{Prefs: ...} here I assume?
```

Comment:
That sounds right, if we're not kicking this update back to the old full netmap processing. It doesn't even need to modify the prefs because `applySysPolicy` will do that.